graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/config/DXGraphQLConfig.java

```diff
@@ -39,13 +39,16 @@ public class DXGraphQLConfig implements ManagedServiceFactory {
     private final static String TYPE_PREFIX = "type.";
 
     private final static String CORS_ORIGINS = "http.cors.allow-origin";
+    private final static String NODE_LIMIT = "graphql.fields.node.limit";
 
     private Map<String, List<String>> keysByPid = new HashMap<>();
     private Map<String, String> permissions = new HashMap<>();
 
     private Set<String> corsOrigins = new HashSet<>();
     private Map<String, Set<String>> corsOriginByPid = new HashMap<>();
 
+    private int nodeLimit = 5000;
```

Comment:
This is a config Factory, so it's supporting multiple configs, are we ok with that, in case multiple configs tries to provide the limit ?
I doubt we want to support multiple config providing the limit, but it's actually possible regarding current implem, just asking.

One solution I see would be to check that the limit property is coming from the **-default.cfg**

Reply:
Should not be a problem but I have added some defensive code against bad values